src/content/getting-started/secure-your-app/_index.md

```diff
@@ -1,29 +1,159 @@
 title: Secure your app
-description: Learn how to secure and protect your application on the Giant Swarm platform.
 weight: 70
-last_review_date: 2024-10-08
+last_review_date: 2024-11-12
+menu:
+  principal:
+    parent: getting-started
+    identifier: getting-started-security
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
   - How do I observe the platform metrics and logs for my application?
   - What do I need to do to observe the platform metrics and logs for my application?
 ---
 
-<Introduction>
+Giant Swarm platform is secured by default. The workload clusters are configured with security best practices in mind. The core functionality is built based on a policy enforcement engine. The `Kubernetes` project comes with [`Pod Security Standards (PSS)`](https://kubernetes.io/docs/concepts/security/pod-security-standards/) to ensure that the pods are running with the least privilege. The main problem of this technology is settings are cluster-wide and it's hard to create exceptions for specific workloads.
+
+For that reason, the platform comes with an extended policy engine, `Kyverno`. A part of enforce the common pod security standards, your clusters have additional policies which can be customized to your needs. Your developers can create exceptions for their workloads without compromising the security of the cluster.
+
+In this guide, you will learn how to make your application compliant with the secure baseline and how create policy exceptions when needed.
 
 ## Requirements
 
-<requirements>
+Before start, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}).
+
+As a next step, you need to deploy an application, here the `hello-world` application, to the workload cluster. For explanation on how to deploy the application please read the [previous step]({{< relref "/getting-started/install-an-application" >}}).
+
+## Step 1: Understand the security baseline
+
+By default the clusters come with a `restricted` set of policies. You can read more about which policies are part of the `restricted` set in the [our documentation]({{< relref "/vintage/advanced/security/security-policy-enforcement/#default-policies" >}}). The `restricted` set of policies are applied to all the namespaces in the cluster.
+
+As brief summary, the `restricted` set of policies includes:
+
+- Containers can only run with `non-root` users
+- Not allowing privileged containers
+- Not allowing host namespaces
+- Allowing only a set of capabilities (indeed, only `NET_BIND_SERVICE` is allowed)
+- `HostPath` volumes aren't allowed
+- Only default `Seccomp` and `AppArmor` profiles are allowed
+- `ReadOnlyRootFilesystem` is enforced
+
+In case you application needs to use any of the features that aren't allowed by the `restricted` policies, you need to create a policy exception.
```

Comment:
```suggestion
In case your application needs to use any of the features that aren't allowed by the `restricted` policies, you need to create a policy exception.
```